src/locales/en.ts

```diff
@@ -279,6 +281,9 @@ const en = {
     // S10
     "ShatteredExilev2-07": "Shattered Exile",
 
+    // S12
+    springtime: "Springtime",
```

Comment:
some maps only have a "versionless" name in PascalCase, some only have one in lowercase.  This change would break some tags unless translations of the base name were copied to be lowercase.  Open to making PascalCase the default as well, but en.ts has way more in lowercase from what I could see